Add validation for returncode and fix bracket to standard

**Is your feature request related to a problem? Please describe.**
Build with options : -Wall -std=c99 -pedantic -Wstrict-prototypes -Wwrite-strings
two warning appear:

/media/sf_share/cFS/tools/elf2cfetbl/elf2cfetbl.c:134:43: warning: missing braces around initializer [-Wmissing-braces]
 union Elf_Shdr SectionHeaderStringTable = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
                                           ^
                                            {                            }
/media/sf_share/cFS/tools/elf2cfetbl/elf2cfetbl.c: In function ‘OpenSrcFile’:
/media/sf_share/cFS/tools/elf2cfetbl/elf2cfetbl.c:1357:9: warning: variable ‘RtnCode’ set but not used [-Wunused-but-set-variable]
     int RtnCode;
         ^~~~~~~

**Describe the solution you'd like**
First issues, add second bracket:
union Elf_Shdr SectionHeaderStringTable = {{0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }};

second issue:
Add validation for return code.

**Requester Info**
Anh Van, NASA Goddard

--- a/elf2cfetbl.c
+++ b/elf2cfetbl.c
@@ -131,7 +131,7 @@ CFE_TBL_File_Hdr_t  TableHeader;
 
 union Elf_Ehdr ElfHeader;
 union Elf_Shdr **SectionHeaderPtrs = NULL;
-union Elf_Shdr SectionHeaderStringTable = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+union Elf_Shdr SectionHeaderStringTable = {{ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }};
 int64 SectionHeaderStringTableDataOffset = 0;
 char  **SectionNamePtrs = NULL;
 
@@ -1367,11 +1367,18 @@ int32 OpenSrcFile(void)
 
     /* Obtain time of object file's last modification */
     RtnCode = stat(SrcFilename, &SrcFileStats);
+    if (RtnCode == 0)
+    {
+        SrcFileTimeInScEpoch = SrcFileStats.st_mtime + EpochDelta;
 
-    SrcFileTimeInScEpoch = SrcFileStats.st_mtime + EpochDelta;
-
-    if (Verbose) printf("Original Source File Modification Time: %s\n", ctime(&SrcFileStats.st_mtime));
-    if (Verbose) printf("Source File Modification Time in Seconds since S/C Epoch: %ld (0x%08lX)\n", SrcFileTimeInScEpoch, SrcFileTimeInScEpoch);
+        if (Verbose) printf("Original Source File Modification Time: %s\n", ctime(&SrcFileStats.st_mtime));
+        if (Verbose) printf("Source File Modification Time in Seconds since S/C Epoch: %ld (0x%08lX)\n", SrcFileTimeInScEpoch, SrcFileTimeInScEpoch);
+    }
+    else 
+    {
+        if (Verbose) printf("Unable to get modification time from %s", SrcFilename);
+        SrcFileTimeInScEpoch = 0;
+    }
 
     return SUCCESS;
 }

